src/api-service/__app__/onefuzzlib/azure/queue.py

```diff
@@ -136,6 +136,8 @@ def remove_first_message(name: QueueNameType, storage_type: StorageType) -> bool
                 return True
         except ResourceNotFoundError:
             return False
+        except TimeoutError:
```

Comment:
How does this relate to the other changes?

Reply:
Ah, I see the commit message. Suggest reverting, splitting into a new PR if appropriate.

Reply:
Removed